### PR TITLE
Updated wiki auto hyper links again

### PR DESCRIPTION
**Pull request by EmosewaMC**
The way these links are created is different than explorer.lu.  This has been fixed with this PR and they correctly generate the hyper links to the wiki.  Tested all pages on mac

### CODE REVIEW

**Comment by Xiphoseer** [comment]
Do you want to try fixing this in conf.py directly? I.e

- add a regex for `(.*)<([A-Za-z_]+)>`
- check if that matches
- if yes, take the first match group (after group 0 for full match) as name, trim whitespace and take the second matvh to build the link
- if no, just do what it does right now, maybe replacing spaces with underscores

**Comment by EmosewaMC** [comment]
> Do you want to try fixing this in conf.py directly? I.e
> 
> * add a regex for `(.*)<([A-Za-z_]+)>`
> * check if that matches
> * if yes, take the first match group (after group 0 for full match) as name, trim whitespace and take the second matvh to build the link
> * if no, just do what it does right now, maybe replacing spaces with underscores

yeah sure - i'll give this a go

**Comment by EmosewaMC** [comment]
Now that I understand Regex in python, I can actually work on this

File: docs/components/102-commendation-vendor.rst [hunk 1]
```diff
@@ -6,5 +6,4 @@ lost or deleted faction items for faction tokens. The only
 object this is attached to is
 :lot:`Honor Accolade -- Commendation Vendor <13806>`.
 
-More information on him may be found here:
-`Honor Accolade <http://legouniverse.wikia.com/wiki/Honor_Accolade>`_
+More information on may be found here: :wiki:`Honor Accolade <Honor_Accolade>`
```
Comment (Xiphoseer) [inline docs/components/102-commendation-vendor.rst:9]: Note: we could probably auto-normalize spaces to underscores in these cases

File: docs/game-mechanics/kit-factions.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@ Kit Factions
 
 The game has four so-called *Kit Factions*: **Assembly**, **Paradox**, **Sentinels**
 and **Venture League**. This has nothing to do with the :doc:`../database/Factions` table, which
-specifies which objects can attack each other. The wiki article on :wiki:`Factions` describes their
+specifies which objects can attack each other. The wiki article on :wiki:`Factions <Factions>` describes their
```
Comment (Xiphoseer) [inline docs/game-mechanics/kit-factions.rst:6]: This change isn't needed anymore afaict

**Review by Xiphoseer — APPROVED** [review]
This is an improvement as-is with two optional things to improve. Please tell if you prefer to add them in here or merge and maybe do that as a follow up.

**Comment by EmosewaMC** [comment]
Would rather improve them all in one go - do it later means put a todo and forget about it :P

File: docs/components/102-commendation-vendor.rst (hunk 1 above)
Comment (EmosewaMC) [reply docs/components/102-commendation-vendor.rst:9]: How would I go about doing this?  just .replace("_", " ") ?

Comment (Xiphoseer) [reply docs/components/102-commendation-vendor.rst:9]: The other way around, but yeah. Just `.replace(" ", "_") to turn a title into the url suffix.

**Review by Xiphoseer — APPROVED** [review]

**Pull request merged**